<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCA Model</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Variable Available</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="VlistWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>240</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>240</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>X Selected Variables</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="XlistWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="XpushButton">
         <property name="text">
          <string>Add to X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="XradioButton">
         <property name="text">
          <string>Rest To X</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Component Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Preprocessing</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="centercheckBox">
            <property name="text">
             <string>Center</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="scalecheckBox">
            <property name="text">
             <string>Scale</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="snvcheckBox">
            <property name="text">
             <string>SNV</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Algorithm</string>
         </property>
         <widget class="QRadioButton" name="nipalsradioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>71</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Nipals</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="svdradioButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>71</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>SVD</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
